.github/workflows/5-step.yml

```
@@ -56,7 +56,7 @@ jobs:
 
```

```suggestion
<h2 hidden>Game over! Want to play again?! Just click refresh. 🧑‍🚀!</h2>
```
Let's make it a bit more friendly. 🤓
```